src/views/catalog/customize/components/CustomizeForms/useCustomizeFormSubmit.ts

```diff
@@ -178,6 +178,12 @@ export const useCustomizeFormSubmit = ({
     }
   };
 
+  React.useEffect(() => {
+    if (templateError) {
+      setTemplateError(undefined);
+    }
```

Comment:
I think you need to disable the eslint warning here

`    // eslint-disable-next-line react-hooks/exhaustive-deps`

Reply:
Ok, thank you :)